### PR TITLE
Add  Gledopto 'GL-SD-301P'

### PR DESCRIPTION
**Pull request by Aogu181**
Add GL-SD-301P
Model : GL-SD-001
Vendor : Gledopto
Description : Zigbee triac AC dimmer

### CODE REVIEW

File: src/devices/gledopto.ts [hunk 1]
```diff
@@ -969,6 +969,14 @@ const definitions: DefinitionWithExtend[] = [
         description: 'Filament LED light bulb E27 G95 7W pro',
         extend: [gledoptoLight({colorTemp: {range: [158, 495]}})],
     },
+    {
+        zigbeeModel: ['GL-SD-301P'],
+        model: 'GL-SD-301P',
+        vendor: 'Gledopto',
+        description: 'Zigbee triac AC dimmer',
+        extend: [gledoptoLight({})],
+        meta: {disableDefaultResponse: true},
```
Comment (Koenkk) [inline src/devices/gledopto.ts:978]: Is this line really needed?

Comment (Aogu181) [reply src/devices/gledopto.ts:978]: yes, I am an employee of this company, and my boss told me to write this. 
Customers require the Z2M to support 'GL-SD-301P'

Comment (Koenkk) [reply src/devices/gledopto.ts:978]: I meant the line `meta: {disableDefaultResponse: true},`, if you remove it, does the device it still work?

Comment (Aogu181) [reply src/devices/gledopto.ts:978]: > I meant the line `meta: {disableDefaultResponse: true},`, if you remove it, does the device it still work?

yes.

**Comment by Aogu181** [comment]
I'm launching a new PR

**Pull request closed without merging**